icu4c/source/common/ucurr.cpp

```diff
@@ -805,21 +796,19 @@ ucurr_getPluralName(const char16_t* currency,
     rb = ures_getByKey(rb, CURRENCYPLURALS, rb, &ec2);
```

Comment:
What about this one, is there any reason for why it shouldn't use `LocalUResourceBundlePointer` too?

Reply:
I talk with Markus about this. ures_getByKey or ures_getByKeyWithFallback  take the third parameter as an optional fill in, in this case, the third parameter is rb which mean it will put the result into rb without allocate additional memory. so the line of LocalUResourceBundlePointer curr will put that pointer into a LocalUResourceBundlePointer.